proto/scene-format.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package turbocow;
+
+message Scene {
+	enum MaterialType {
+		SOLID = 0;
+		GLASS = 1;
+		// other matherials
+	}
+	message Vector3 {
+		float x = 2;
+		float y = 3;
+		float z = 4;
+	}
+	message Transform {
+		int32 id = 5;
+		Vector3 translation = 6;
```

Comment:
`position` is a better name I think

Reply:
`Position` does not represent the rotation and scale, so maybe something like `state` unless `position` is common in CG.

Reply:
I mean it is better to rename `translation` to `position`. `position` is more commonly used to describe this parameter (at least from what I saw), but it is just a suggestion and both names describe parameter perfectly

Reply:
> I mean it is better to rename `translation` to `position`. `position` is more commonly used to describe this parameter (at least from what I saw), but it is just a suggestion and both names describe parameter perfectly

Ah, okay. I thought you were talking about Transform. Sorry)